MovieLibrary/MovieList.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+class MovieStorage: ObservableObject {
+    @Published var selectedGenre : Genre = .fantasy
+    @Published var sortingOption : SortingOption = .title
+    
+    let movies = [
```

Comment:
I think this property should be private, cause you don't use it outside of class anywhere :)

Reply:
These properties shouldn't be private because they are used in ContentView